nautobot_chatops/models.py

```diff
@@ -142,7 +151,30 @@ class Meta:
         unique_together = [["platform", "token"]]
 
 
+class ChatOpsAccountLink(PrimaryModel):
```

Comment:
`development/mattermost/nautobot_bootstrap.py` can be updated with value linkink nautobot admin to mattermost admin